src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'categories/CHECK_STATUS';
+
+const INITIAL_STATE = 0;
+
+const categoryReducer = (state = INITIAL_STATE, action = {}) => {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return 'Under Construction.';
+    default:
+      return state;
+  }
+};
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
```

Comment:
- [ ] [OPTIONAL] Kindly use `createSlice` from `@reduxjs/toolkit` to create reducers instead of creating them manually.

Reply:
Thank you for suggesting the use of createSlice from @reduxjs/toolkit to create reducers. I will consider incorporating it into my codebase. Appreciate your feedback. 😁